Cargo.toml

```diff
@@ -32,136 +32,137 @@ parity-scale-codec = { version = "3.0.0", default-features = false }
-cumulus-relay-chain-inprocess-interface = { version = "0.10.0", default-features = false }
-cumulus-relay-chain-interface           = { version = "0.10.0" }
-cumulus-relay-chain-minimal-node        = { version = "0.10.0", default-features = false }
-cumulus-relay-chain-rpc-interface       = { version = "0.10.0", default-features = false }
-pallet-collator-selection               = { version = "12.0.0", default-features = false }
-parachains-common                       = { version = "10.0.0", default-features = false }
-staging-parachain-info                  = { version = "0.10.0", default-features = false }
+cumulus-client-cli                      = { version = "0.11.0" }
+cumulus-client-collator                 = { version = "0.11.0" }
+cumulus-client-consensus-aura           = { version = "0.11.0" }
+cumulus-client-consensus-common         = { version = "0.11.0" }
+cumulus-client-consensus-proposer       = { version = "0.11.0" }
+cumulus-client-consensus-relay-chain    = { version = "0.11.0" }
+cumulus-client-network                  = { version = "0.11.0", default-features = false }
+cumulus-client-parachain-inherent       = { version = "0.5.0" }
+cumulus-client-service                  = { version = "0.11.0", default-features = false }
+cumulus-pallet-aura-ext                 = { version = "0.11.0", default-features = false }
+cumulus-pallet-parachain-system         = { version = "0.11.0", default-features = false }
+cumulus-pallet-session-benchmarking     = { version = "13.0.0", default-features = false }
+cumulus-pallet-xcm                      = { version = "0.11.0", default-features = false }
+cumulus-pallet-xcmp-queue               = { version = "0.11.0", default-features = false }
+cumulus-primitives-aura                 = { version = "0.11.0", default-features = false }
+cumulus-primitives-core                 = { version = "0.11.0", default-features = false }
+cumulus-primitives-parachain-inherent   = { version = "0.11.0", default-features = false }
+cumulus-primitives-utility              = { version = "0.11.0", default-features = false }
+cumulus-relay-chain-inprocess-interface = { version = "0.11.0", default-features = false }
+cumulus-relay-chain-interface           = { version = "0.11.0" }
+cumulus-relay-chain-minimal-node        = { version = "0.11.0", default-features = false }
+cumulus-relay-chain-rpc-interface       = { version = "0.11.0", default-features = false }
+pallet-collator-selection               = { version = "13.0.0", default-features = false }
+parachains-common                       = { version = "11.0.0", default-features = false }
+staging-parachain-info                  = { version = "0.11.0", default-features = false }
 
 # ORML
-orml-vesting = { version = "0.9.1", default-features = false }
+orml-vesting = { git = "https://github.com/ajuna-network/open-runtime-module-library.git", branch = "cl/polkadot-v1.10.0-Cargo.toml", default-features = false }
```

Comment:
Upstream has not released to crates.io yet, but I filed a PR there: https://github.com/open-web3-stack/open-runtime-module-library/pull/994